src/components/Select/Select.tsx

```diff
@@ -13,14 +16,22 @@ export interface SelectProps {
   children?: React.ReactNode;
   /** When true, disables the select */
   disabled?: boolean;
-  /** ID of associated HelperText */
+  /** Helper for modifiers array; adds 'errored' styling */
+  errored?: boolean;
+  /** Populates the HelperErrorText for error state */
+  errorText?: string;
+  /** DEPRECATED - ID of associated HelperText */
   helperTextId?: string;
+  /** Populates the HelperErrorText for standard state */
+  helperText?: string;
   /** ID that other components can cross reference for accessibility purposes */
   id?: string;
-  /** Attribute indicating that an option with a non-empty string value must be selected */
-  isRequired: boolean;
-  /** ID of associated label */
+  /** DEPRECATED - Attribute indicating that an option with a non-empty string value must be selected */
```

Comment:
I think these labels are great to have, but maybe we can break the alphabetical list convention and put all the deprecated props at the bottom of the list and have one comment such as, "DEPRECATED - the following props are still available to use but are deprecated and will be removed in a future release", or similar.

Reply:
This reminds me: maybe we can have a `console.warn` message if any of the deprecated props are used that says something like "These props used in the Select component will be deprecated in a future release: ...... Check the documentation for updated props to use instead."

Reply:
@EdwinGuzman I will add a `console.warn` message if/when a deprecated prop is used.  I like that a lot.  But for Storybook, what about hiding the props in the `ArgsTable` and including a note about the deprecated props below the ArgsTable?

Reply:
Yes, hiding the props in Storybook and adding a note are good ideas.

Reply:
Done.